### ISSUE TITLE
Docker environment variable for max upload size?

### ISSUE THREAD

**Issue by mindfuucker** [proofrock/pupcloud#10]
Increasing the maximum upload size in Docker is quite cumbersome at the moment as my docker-compose.yml file shows:
```
version: "3"
services:
  pupcloud:
    image: germanorizzo/pupcloud:latest
    user: 1003:1003
    ports:
      - "17178:17178"
    restart: unless-stopped
    entrypoint: ["/pupcloud", "-r", "/data", "--max-upload-size", "1000"]
    environment:
      PID: 1003
      PGID: 1003  
    volumes:
      - /home/user/music:/data
```
If you know how entry points work it quite doable but I suspect users would want an easier method. Or just having documentation about the entrypoint with an example might be sufficient for the moment.

**Comment by proofrock** [comment]
If I do that for `--max-upload-size` I should do it for every parameter, and it could be impossible to di it for example for  (multiple) `--share-profile`. I'll think about it, I may end up leaving the sharing part out of it.

**Comment by proofrock** [comment]
See commit [bf245f4](https://github.com/proofrock/pupcloud/commit/bf245f478dadfd868f888b10efa1b3ffa8b01768)

**Comment by proofrock** [comment]
Released in v0.8.0
